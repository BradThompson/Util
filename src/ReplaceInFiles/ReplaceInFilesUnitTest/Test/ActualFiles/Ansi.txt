This is an Ansi encoded file.
