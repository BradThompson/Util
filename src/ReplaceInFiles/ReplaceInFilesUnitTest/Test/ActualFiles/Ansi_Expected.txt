This is an Ansi test file.
